trade-finance-manager-api/src/v1/controllers/auth/auth-provider-getAuthCodeUrl.api-test.js

```diff
@@ -0,0 +1,125 @@
+const AuthProvider = require('./auth-provider');
```

Comment:
auth-provider - file originated from Microsoft example and is meant abstract npm `@azure/msal-node`.
auth-service - is try to use service pattern and put helper functions that are not reused there.

Names doesn't match because functionality is not the same.

Will rename to make these unit tests

Reply:
Oh no I mean `auth-provider-getAuthCodeUrl` vs `auth-provider-get-auth-code-url` sorry!

Thanks for the rename -- will resolve once ive seen it